src/mappersistor.ts

```diff
@@ -0,0 +1,89 @@
+import * as fs from 'fs';
+import {Persistor} from './persistor'
+
+/** 
+ * Persists maps that hold number-keys and values of type V in the filePath.
+ */
+export class MapPersistor<V> implements Persistor<V>{
+
+    /** 
+     * Creates an instance of MapPersistor<V>
+     * 
+     * @param {string} filePath The path at which save-data should be saved and loaded
+     */
+    public constructor(private readonly filePath: string) {
+        console.log('Persistor: In working order');
+    }
+
+    /**
+     * Loads the saved map, if there is a save-file.
+     * 
+     * If there is no save-file a new, empty map is created.
+     * 
+     * @returns {[Map<number, V>, number]} A promise for an Array holding the loaded map, as well as the number right above the highest key
+     */
+    public async loadData(): Promise<[Map<number, V>, number]> {
```

Comment:
It is better to work with objects/classes instead of simple arrays. In case of arrays, one has to know, what is `[0]`, what is `[1]`, ....